src/instructlab/model/serve_backend.py

```diff
@@ -23,6 +28,55 @@ def signal_handler(num_signal, __):
     sys.exit(0)
 
 
+def get_gpu_count() -> int:
```

Comment:
I think pytorch already provides this as a function: 
INFO 02-03 01:15:49 launcher.py:27] Route: /v1/embeddings, Methods: POST

Reply:
removed it